frontend/src/components/organisms/Table/style.css

```diff
@@ -56,6 +56,7 @@
 
 .datapass-table th,
 td {
+  font-weight: 400;
```

Comment:
this line conflicts with master branch and I was unable to rebase it myself as I do not understand the purpose of this. This seems incompatible with 

https://github.com/betagouv/datapass/commit/6707390aa275cd8d83c3765c0d4e20e074c1ef4b#diff-22a9fe5eaeed98d0d5e8c4e296b3fabc59484cd3dc8b77b179ff80046d7240cfR113-R115